libs/builders/src/semantic-release/lib/get-analyze-commits-options.ts

```diff
@@ -11,7 +11,7 @@ export function getAnalyzeCommitsOptions(project: string, mode: 'independent' |
     parserOpts = { headerPattern: new RegExp(`^(\\w*)(?:\\((${project}|\\*|deps)\\))?: (.*)$`) };
     releaseRules = releaseRules.map((rule) => ({ ...rule, scope: project }));
   } else {
-    parserOpts = { headerPattern: new RegExp(`^(\\w*)(?:\\((\\w*)\\))?: (.*)$`) };
+    parserOpts = { headerPattern: new RegExp(`^(.*)(?:\\((\\w*)\\))?: (.*)$`) };
```

Comment:
The first part of the regex is for the commit type, such as `feat`, `fix` and so on, and will always be a word. Actually the fix should be in the scope part:

```suggestion
    parserOpts = { headerPattern: new RegExp(`^(\\w*)(?:\\(([^:]*)\\))?: (.*)$`) };
```

You can test it here https://regex101.com/r/efCmKF/1/

Reply:
Good catch. I shouldn't have written that at 1 AM.